.github/workflows/deploy-dev.yml

```
@@ -48,7 +45,10 @@ jobs:
       - name: Install dependencies
         run: |
           echo "Installing root dependencies..."
-          npm ci
+          npm install
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)